posts/app.py

```diff
@@ -204,7 +204,8 @@ def upload_new_post_to_db(post, collection):
     # post is valid, add on timestamp, upload files, insert into db
     post['created_at'] = generate_timestamp()
     post['files'] = [
-        upload_file_to_cloud(file) for file in post['files']]
+        upload_file_to_cloud(file) for file in post['files']
+        if file.filename != '']
```

Comment:
Would it be better to handle this check in pageserve?

Reply:
Yeah, sorry, fixed in https://github.com/knative-portability/large-events/pull/74/commits/71bd359d2ff3be1c085d33837db1225be4d4245a